src/advanced/node-management.md

```diff
@@ -559,258 +556,193 @@ Response is a JSON object with the following fields:
       },
-      "location": {
-        "block_height": "18",
-        "position_in_block": "261"
+    "location": {
+      "block_height": 11,
+      "position_in_block": 0
       },
-      "location_proof": {
-        "left": "07e641264ac4646495c54a379a5943bf88785bcf30a0b4c13f47d1e2e62b343d",
-        "right": {
-          "left": {
-            "left": {
-              "left": {
-                "left": {
-                  "left": {
-                    "left": "78044db9c2713a11c2fe7bb66f27665b6ca0ecbb9e61e09381534d449c4c24c4",
-                    "right": {
-                      "left": {
-                        "left": "f62e6a4d8b9c2c0cfb31ea599e08e6e3fe2337169ce07008d91390958e0613d4",
-                        "right": {
-                          "val": "f6415994136527a24d022595ec0d40f51e2a0c4230a34792a5203df779e3ffaf"
-                        }
-                      },
-                      "right": "daee36b5b7c24a831a62539d534c56e4f234a83ce83876fee48193d8eefabfb2"
-                    }
-                  },
-                  "right": "df8bb7e697e6eb7828975b67f64a77af06379435f4330f0cb0b7a21d18b616db"
-                },
-                "right": "59ab567cf0d1c09c5050680ea4ed4650238023666b0e5d90855d82dc83d1f982"
-              },
-              "right": "cf5fa45701ba6ae795da58a504eb1608075d7bbedf05c8f6f448aad4dbd03968"
-            },
-            "right": "eb583ba724a3b371c243f1268dfca2929c704fe0546875e5dfeb90860ac3533f"
-          },
-          "right": "bea1bae302dbf975cafa064ecfbc39f2cdcba5fe7ffddb2208c060cd9778c483"
-        }
+    "location_proof": {
+      "val": "2f23541b10b258dfc80693ed1bf6ed6f53ccf8908047f7d33e0fec4f29a4a613"
       },
-      "status": {
-        "type": "success"
+    "status": {
+      "type": "success"
       }
     }
     ```
+<!-- markdownlint-enable MD013 -->
 
-### Block by height
+### Block by Height
 
 ```none
 GET {explorer_base_path}/block?height={height}
 ```
 
-Returns the content for a block of a specific height.
+Returns content of the block at a specific height.
 
 #### Parameters
 
 - **height**: integer  
-  The height of the desired block
+  Height of the desired block.
 
 #### Response
 
 A JSON object with the following fields:
 
 - **block**: BlockHeader  
-  The header of the specified block
-- **precommits**: Array<Precommit\>  
-  The list of 'Precommit' messages supporting the block.
+  Header of the specified block.
+- **precommits**: Array<Hash\>  
+  List of hashes of the 'Precommit' messages supporting the block.
 - **txs**: Array<SerializedTransaction\>  
-  The list of the transactions included into the block
+  List of the transactions included into the block.
+- **time**: time object
```

Comment:
I would add some description of that object.
What is the time format? Is it time with time-zone?

Reply:
This has already been done in the "Types section" above https://github.com/exonum/exonum-doc/blob/5b96ff8181ee14d5bab2b295ea23964ef6b0bd4e/src/advanced/node-management.md#time